### PR TITLE
Split utils

### PR DESCRIPTION
**Pull request by gsvr**
split utils into generic and project-specific files; utils.js & utils_project.js

### CODE REVIEW

**Comment by gsvr** [comment]
@imsickofmaps Please review.

File: src/utils_project.js [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+var Q = require('q');
+
+
+// Project utils libraty
+go.utils_project = {
+
+// TIMEOUT HELPERS
+
+    timed_out: function(im) {
+        var no_redirects = [
```
Comment (imsickofmaps) [inline src/utils_project.js]: Let's put the list in the config and move it to utils

File: src/utils_project.js [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+        return Q()
+            .then(function(q_response) {
+                return msisdn === '082333';
+            });
+    },
+
+
+// SUBSCRIPTION HELPERS
+
+    get_active_subscriptions_by_identity_id: function(identity_id, im) {
```
Comment (imsickofmaps) [inline src/utils_project.js]: can we evaluate which of these are really project focused?

**Comment by imsickofmaps** [comment]
Make changes and I'll do a second pass

**Comment by gsvr** [comment]
@imsickofmaps Second pass please

**Comment by imsickofmaps** [comment]
:+1:

**Pull request merged**